src/app/system/dashboard.view.html

```diff
@@ -82,7 +82,7 @@ <h2 class="m-b-xs" ng-bind="(vm.supervisor.processes['minemeld-engine']['childre
         <div class="dashboard-label-text">API</div>
       </div>
       <div class="col-xs-10 text-center dashboard-main-statistic">
-        <h2 class="m-b-xs" ng-bind="(vm.supervisor.processes['minemeld-web']['children'])||N/A"></h2>
+        <h2 class="m-b-xs" ng-bind="(vm.supervisor.processes['minemeld-engine']['children'])||'N/A'"></h2>
```

Comment:
I think this is a copy & paste error, it should be minemeld-web not minemeld-engine

Reply:
Updated it back to 'minemeld-web'